### ISSUE TITLE
Keycloak Container never starts

### ISSUE THREAD

**Issue by boev** [dasniko/testcontainers-keycloak#2]
Hello,

I have a strange issue with the container. It never starts on time, where previously and on another machine it works nicely. I get the error:

11:40:03.236 [main] INFO org.testcontainers.containers.wait.strategy.HttpWaitStrategy - /funny_mcnulty: Waiting for 120 seconds for URL: http://localhost:32773/auth
11:40:05.021 [tc-okhttp-stream-1189259123] INFO 🐳 [jboss/keycloak:7.0.0] - STDOUT: Added 'admin' to '/opt/jboss/keycloak/standalone/configuration/keycloak-add-user.json', restart server to load user
11:42:03.240 [main] DEBUG 🐳 [jboss/keycloak:7.0.0] - Wait strategy threw an exception
org.testcontainers.containers.ContainerLaunchException: Timed out waiting for URL to be accessible (http://localhost:32773/auth should return HTTP 200)

I pruned everything (network, volumes, containers)  uninstalled and installed docker a new. Same thing. I am able to start other test containers successfully, actually any I tried...mysql, redis and so on...
The keycloak was previously accessible and the same code does work on another machine. Increasing the timeout to 5 minutes solves the problem but it seems like keycloak just hangs for a long time not starting and then it suddenly starts. Any ideas?

**Comment by boev** [comment]
It was a networking issue with VPN and VirtualBox. Closing the issue.

**Comment by AntonBozhinov** [comment]
I'm having the same issue, but i'm not connected to a VPN or running in a virtual machine :(

**Comment by sventorben** [comment]
I have the same problem with my setup. @boev What exactly caused the issue with the VPN and how did you fix it?

**Comment by paultid** [comment]
I am seeing a similar issue on Docker for Windows.   I get:

` Added 'admin' to '/opt/jboss/keycloak/standalone/configuration/keycloak-add-user.json', restart server to load user`

Then after more than 2 minutes, keycloak proceeds to start up and everything is fine. This happens even when starting keycloak in docker from the command line, so not related to the Keycloak Container.  This long delay causes the ContainerLaunchException. 

Has anyone else found a solution?

**Comment by sventorben** [comment]
@paultid I was able to "fix" it by setting the `DB_VENDOR` environment variable explicitly. 

I experienced that issue on my laptop, but only when beeing connected to certain networks. I assume it has something to do with dns settings/lookups. Keycloak [entrypoint.sh](https://github.com/keycloak/keycloak-containers/blob/master/server/tools/docker-entrypoint.sh#L122) is trying to determine the database address here, which seems to cause the issue.
